earthaccess/auth.py

```diff
@@ -25,6 +25,24 @@
 logger = logging.getLogger(__name__)
 
 
+def netrc_path() -> Path:
+    """Return the path of the `.netrc` file.
+
+    The path may or may not exist.
+
+    See [the `.netrc` file](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html).
+
+    Returns:
+        `Path` of the `NETRC` environment variable, if the value is non-empty;
+        otherwise, the path of the platform-specific default location:
+        `~/_netrc` on Windows systems, `~/.netrc` on non-Windows systems.
+    """
+    sys_netrc_name = "_netrc" if platform.system() == "Windows" else ".netrc"
```

Comment:
This distinction is a good catch! I've never used the library on Windows, I wonder if someone could share an example with us.

Reply:
Search for `_netrc` in the `requests` code shown here: https://requests.readthedocs.io/en/latest/_modules/requests/utils/#get_netrc_auth